js/upload/src/Upload.svelte

```diff
@@ -17,6 +13,9 @@
 	export let disable_click = false;
 	export let root: string;
 
+	// Needed for wasm support
+	const upload_fn = getContext<typeof upload_files>("upload_files");
```

Comment:
@pngwn I followed your recommendation about adding back getContext. Should be good for another look. Thanks!